libs/ui/src/components/icon/assets/receive-small.svg

```diff
@@ -1,10 +1,10 @@
-<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_7019_58711)">
-<path d="M7.97461 16.7197C7.55762 16.7197 7.25684 17.0342 7.25684 17.458C7.25684 17.8818 7.55762 18.1963 7.97461 18.1963H17.0049C17.4287 18.1963 17.7363 17.8818 17.7363 17.458C17.7363 17.0342 17.4287 16.7197 17.0049 16.7197H12.6572C12.8076 16.6924 12.9443 16.6104 13.0537 16.4941L17.4219 12.167C17.5859 12.0029 17.668 11.8184 17.668 11.6201C17.668 11.1963 17.3604 10.8887 16.9502 10.8887C16.7383 10.8887 16.5469 10.9775 16.4102 11.1143L15.002 12.4951L13.1768 14.498L13.2451 13.0625V6.69824C13.2451 6.24707 12.9375 5.93945 12.5 5.93945C12.0557 5.93945 11.748 6.24707 11.748 6.69824V13.0625L11.8164 14.5049L9.99121 12.4951L8.58984 11.1143C8.44629 10.9707 8.26172 10.8887 8.0498 10.8887C7.63281 10.8887 7.33203 11.1963 7.33203 11.6201C7.33203 11.8184 7.40723 12.0029 7.57129 12.167L11.9395 16.4941C12.0488 16.6104 12.1855 16.6924 12.3359 16.7197H7.97461Z" fill="#ED3D25"/>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_7067_92532)">
+<path d="M7.47461 16.7197C7.05762 16.7197 6.75684 17.0342 6.75684 17.458C6.75684 17.8818 7.05762 18.1963 7.47461 18.1963H16.5049C16.9287 18.1963 17.2363 17.8818 17.2363 17.458C17.2363 17.0342 16.9287 16.7197 16.5049 16.7197H12.1572C12.3076 16.6924 12.4443 16.6104 12.5537 16.4941L16.9219 12.167C17.0859 12.0029 17.168 11.8184 17.168 11.6201C17.168 11.1963 16.8604 10.8887 16.4502 10.8887C16.2383 10.8887 16.0469 10.9775 15.9102 11.1143L14.502 12.4951L12.6768 14.498L12.7451 13.0625V6.69824C12.7451 6.24707 12.4375 5.93945 12 5.93945C11.5557 5.93945 11.248 6.24707 11.248 6.69824V13.0625L11.3164 14.5049L9.49121 12.4951L8.08984 11.1143C7.94629 10.9707 7.76172 10.8887 7.5498 10.8887C7.13281 10.8887 6.83203 11.1963 6.83203 11.6201C6.83203 11.8184 6.90723 12.0029 7.07129 12.167L11.4395 16.4941C11.5488 16.6104 11.6855 16.6924 11.8359 16.7197H7.47461Z" fill="#14BA30"/>
 </g>
 <defs>
-<clipPath id="clip0_7019_58711">
-<rect width="24" height="24" fill="white" transform="translate(0.5)"/>
```

Comment:
is this correct? you have changed the color of the already existing SVG - this may affect `CollectiblesWidget`.  (you could use `color` prop if you need the same svg but with another color)

Reply:
yes we have two different SVG